client/src/pages/ApplicationsPage/ApplicationsPage.tsx

```diff
@@ -5,15 +5,32 @@ import { useAppSelector } from '../../app/hooks';
 import { IApplication } from '../../../types/applications';
 import ApplicationDashboard from '../../components/ApplicationDashBoard/ApplicationDashBoard';
 
-const ApplicationsPage = (): JSX.Element => {
+interface Params {
+  userId: string;
+  status?: string;
+  date?: string;
+}
+
+const ApplicationsPage = () => {
   const navigate = useNavigate();
   const [applications, setApplications] = useState<IApplication[]>([]);
```

Comment:
This page seems to be for showing a user their own applications - not other users applications. Would it be better to store this in redux? Local state makes more sense to me when viewing other users application data

Reply:
Similar to the previous question. This will be migrated to Redux.

Reply:
future tickets ✅